docs/topics/develop/patterns/asynchronous-processing.md

```diff
@@ -0,0 +1,90 @@
+
+## Getting Started
+1. [Create a cache in the Momento console](https://console.gomomento.com/caches/create)
+2. [Create a webhook](/topics/webhooks/creating-a-webhook) for the cache. Assign the Webhook Destination to the public facing endpoint.
+3. Add code to the Webhook to process the incoming messages
+```typescript
+import {
+    APIGatewayProxyEvent,
+    APIGatewayProxyResult 
+} from "aws-lambda/trigger/api-gateway-proxy";
+
+import {TopicClient, CredentialProvider} from '@gomomento/sdk';
+
+
+interface MomentoWebhookEvent {
+    cache: string;
+    topic: string;
+    event_timestamp: number;
+    publish_timestamp: number;
+    topic_sequence_number: number;
+    token_id: string;
+    text: string;
+}
+
+const momento = new TopicClient({
+    credentialProvider: CredentialProvider.fromString('<the api key>'),
+})
+
+export const lambdaHandler = async (
+    event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> => {  
+    const webhookEvent: WebhookEvent = JSON.parse(event.body);
+    // simply take the current message, uppercase it, and publish to a new topic
+    await momento.publish(webhookEvent.cache, "topic 2", webhookEvent.text.toUpperCase());
+    return {
+        statusCode: 200,
+        body: JSON.stringify({status : "success"})
+    }
+}
+```
+4. Add a subscriber to this new `topic 2`
+```typescript
+const result = await topicClient.subscribe(cacheName, 'topic 2', {
+    onError: () => {
+        console.error('Received an error from the topic');
+        return;
+    },
+    onItem: (item: TopicItem) => {
+        console.log(`Received uppercase text on topic 'topic-b': ${item.value().toString()}`);
+        return;
+    },
+});
+```
+5. Begin publishing messages to the Topic that the Webhook is listening to. You can use the [Momento SDK](/topics/develop/api-reference) to publish to the topic.
+```typescript
+import {TopicPublish} from '@gomomento/sdk';
+const publishResponse = await topicClient.publish(cacheName, 'topic 1', 'a value');
+if (publishResponse instanceof TopicPublish.Success) {
+    console.log('Value published successfully!');
+} else {
+    console.log(`Error publishing value: ${publishResponse.toString()}`);
+}
+```
+6. The subscriber to `topic 2` should now be receiving the uppercase messages, and logging the messages to the console!
+
+And that is it! This Topics pattern allows for clients to be completely agnostic of the downstream consumer, and allows flexibility down the line for consuming messages from the client. For a more complete example of using Webhooks for event processing, checkout our EventBridge [example](/topics/integrations/lambda-handler), and our [blog post](https://www.gomomento.com/blog/how-to-use-webhooks-and-momento-topics-to-build-a-multi-language-chat-app) about using this pattern to create a multi-language chat application.
```

Comment:
I might add one or two more sentences here. One that said something like "in the future, if your traffic increases and your server-side component outgrows an environment like webhooks+APIG+Lambda, you can replace the server-side with containers or EC2 resources." and one that said "this is the simplest possible example of basic string conversion, but you can imagine more sophisticated processing use cases such as:

* storing to a database
* moderating using a service like blah blah blah
* eventbridge
"